src/script/media/MediaDevicesHandler.test.ts

```diff
@@ -23,113 +23,7 @@ import {MediaDevicesHandler} from './MediaDevicesHandler';
 
 /* yarn test:app --specs media/MediaDevicesHandler --nolegacy */
 describe('MediaDevicesHandler', () => {
```

Comment:
I wouldn't remove the test. You can adapt the test quite easily. The test only counts the number of devices after the filter.

The test describes exactly what you did. You no longer filter by group ID.